package.json

```diff
@@ -11,9 +11,13 @@
     "url": "https://github.com/LionWeb-io/lionweb-repository/issues"
   },
   "scripts": {
-    "build": " npm run build -w packages",
-    "lint": "npm run lint -w packages",
+    "initialize": "npm run clean && npm install && npm run setup && npm run build",
+    "clean": "rm -rf node_modules/ && npm run clean -w packages --if-present",
```

Comment:
what does --if-present means? Does it run the task for all packages that have such task?